### PR TITLE
Add simple hello world fastapi app

### LINKED ISSUE #3: Create hello world
* Create a basic python API app having a hello world endpoint:  `/hello_world`. It will return a JSON like this:
    ```json
    {"data": "Hello world"}
    ```

### PR DESCRIPTION
**Pull request by niedziol**
Closes #3

### CODE REVIEW

File: webhook/tests/test_main.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+from fastapi.testclient import TestClient
+
+from webhook.main import app
+
+client = TestClient(app)
+
+def test_index():
+    response = client.get('/')
+    assert response.status_code == 200
+    assert response.text == '{"data":"Hello World"}'
```
Comment (karpp) [inline webhook/tests/test_main.py:11]: You can do response.json() == {"msg": "Hello World"}

Comment (niedziol) [reply webhook/tests/test_main.py]: Done

File: webhook/app.yaml [hunk 2]
```diff
@@ -1,3 +1,3 @@
 runtime: python38
 service: webhook
-
+entrypoint: uvicorn --port $PORT
```
Comment (mgmacias95) [inline webhook/app.yaml:3]: missing `host` [option](https://www.uvicorn.org/settings/). The default one is `127.0.0.1`, which will make it impossible to receive requests when it's deployed in production.

Comment (niedziol) [reply webhook/app.yaml]: Done

**Pull request merged**